src/guides/v2.4/test/integration/parameterized_data_fixture.md

```diff
@@ -0,0 +1,101 @@
+---
+group: testing
+title: Parameterized Data Fixture
+---
+
+{:.bs-callout-info}
+Parameterized Data Fixture currently only available for Magento Open Source contributors, and will be released for public with Magento Open Source 2.4.5.
```

Comment:
```suggestion
Parameterized Data Fixture is currently only available for Magento Open Source contributors. It will be released for general use with Magento Open Source 2.4.5.
```